src/poojap/PrimeNumberr.java

```diff
@@ -0,0 +1,27 @@
+package poojap;
+
+public class PrimeNumberr {
+
+	void findPrime(int start, int end) {
+
+		System.out.println("Prime Numbers from " + start + " to " + end);
+
+		for (int i = start; i <= end; i++) {
+			int num = i;
+			boolean flag =true;
+			for (int j = 2; j <= num / 2; j++) {
+				if (num % j == 0)
+					flag = false;
+			}
```

Comment:
here, we should break the loop, because, we know flag value update to false only when number is not prime number, then why we will iterate aging for given number.